app.py

```diff
@@ -219,9 +219,49 @@ def geofence() -> Response:
             "dataUpdated": geofence
         }), 200
 
+# Get and Post Search Area
+@app.route('/api/searchArea', methods = ['GET', 'POST'])
+def searchArea():
+    # Get searchArea
+    if request.method == 'GET':
+        searchArea_data = TABLES['searchArea'].all()
+
+        return jsonify(searchArea_data)

```

Comment:
The current return type for this endpoint is a `List[Document]` array. Can you make it return just a `Document` object instead?
```js
// Current response body
[
    {
        "coordinates": [
            {
                "lat": 45,
                "lng": -45
            },
            {
                "lat": 45,
                "lng": -45
            },
            {
                "lat": 45,
                "lng": -45
            }
        ]
    }
]

// Desired response body
{
     "coordinates": [
         {
            "lat": 45,
            "lng": -45
        },
        {
            "lat": 45,
            "lng": -45
        },
        {
            "lat": 45,
            "lng": -45
        },
    ]
}
```